CHANGES.rst

```diff
@@ -36,6 +36,8 @@ Features added
 * #13146: Napoleon: Unify the type preprocessing logic to allow
   Google-style docstrings to use the optional and default keywords.
   Patch by Chris Barrick.
+* Add a subtype ``toc.glob_not_matching`` to a warning of a non-matching glob
+  pattern in TOC.
```

Comment:
Describe why the change is being made, rather than what it does
```suggestion
* Allow supressing warnings from the :rst:dir`toctree` directive when a glob pattern
  doesn't match any documents, via the new ``toc.glob_not_matching`` warning sub-type.
  Patch by Slawek Figiel.
```

Reply:
Sure, I've rephrased the sentence a bit.